content/setup/overview.md

```diff
@@ -7,6 +7,19 @@ menu = "installation"
 toc = true
 +++
 
+# Features support
+
+| Feature/Remote            | GitHub  | GitLab  | BitBucket           | Gogs    |
+|---------------------------|---------|---------|---------------------|---------|
+| Supported version         | --/--   | 8.2+    | --/--               | --/--   |
+| VCS                       | **git** | **git** | **git ( only )**    | **git** |
```

Comment:
I don't use mercurial, but it was my understanding that we only supported git for Bitbucket as well. I see that we have a mercurial plugin, but the webhook values are a bit different for Hg repos. Not wildly different, but different enough to where we wouldn't automatically have working Hg support just by virtue of having an Hg plugin.